src/Executables/CMakeLists.txt

```diff
@@ -3,4 +3,5 @@
 
 add_subdirectory(Benchmark)
 add_subdirectory(DebugPreprocessor)
+add_subdirectory(HelloWorldNoCharm)
```

Comment:
I would prefer to put this in Executables/Examples/HelloWorld/ as I did in #751 to avoid clutter in the Executables directory